requirements/edx/github.txt

```diff
@@ -106,3 +106,5 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.18#egg=xblock-
 # Totem ind xblock (greenhost)
 -e git+https://code.greenhost.net/totem/ind.git#egg=xblock-ind
 
+# Custom video_xblock
+git+https://github.com/raccoongang/xblock-video.git@ginkgo-rg#egg=video_xblock
```

Comment:
Does it a link to branch? I would prefer we add tag into xblock and link on it in the requirements.

Reply:
Updated